stablehlo/dialect/Base.cpp

```diff
@@ -600,5 +600,23 @@ ShapedType createShapedType(ShapedTypeComponents components) {
   return UnrankedTensorType::get(components.getElementType());
 }
 
+// TODO(#1578): Remove this code once all uses of I64DenseArrayOrElements1DAttr
+// have been removed.
+SmallVector<int64_t> getI64Array(Attribute attr) {
+  SmallVector<int64_t> vec;
```

Comment:
Related comment from other PR:
https://github.com/openxla/stablehlo/pull/1905#discussion_r1443310097

Can address in either PR review